corehq/apps/userreports/dbaccessors.py

```diff
@@ -0,0 +1,10 @@
+def get_report_configs_by_data_source(domain, data_source_id):
```

Comment:
that makes sense. I was mainly thinking that `dbacessors` was a couch thing (do you think we would move django model queries there?) and the main advantage was to get a picture of the queries that are dependent on views so that we can have a picture of what those are and hopefully eventually kill them one by one. putting them all in one module doesn't prevent that use case either, it just makes it take a little more headspace to do